lang/python/python3-bottle/Makefile

```diff
@@ -8,20 +8,19 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python3-bottle
-PKG_VERSION:=0.12.16
+PKG_VERSION:=0.12.17
 PKG_RELEASE:=1
 
-PKG_LICENSE:=MIT
-PKG_LICENSE_FILES:=LICENSE
-PKG_CPE_ID:=cpe:/a:bottlepy:bottle
-
-PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
-
 PKG_SOURCE:=bottle-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/b/bottle
-PKG_HASH:=9c310da61e7df2b6ac257d8a90811899ccb3a9743e77e947101072a2e3186726
+PKG_HASH:=e9eaa412a60cc3d42ceb42f58d15864d9ed1b92e9d630b8130c871c5bb16107c
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-bottle-$(PKG_VERSION)
```

Comment:
Is this necessary? There's no python-bottle package. I assume we can get rid of the VARIANT.

Reply:
It was necessary because of PKG_UNPACK, but this one is not necessary. So, I removed it and changed it as you suggested.